cli/Valet/Site.php

```diff
@@ -477,8 +480,11 @@ public function secure($url, $siteConf = null, $caExpireInDays = 368)
 
         $this->files->ensureDirExists($this->nginxPath(), user());
 
-        $this->createCa($caExpireInDays);
-        $this->createCertificate($url, $caExpireInDays);
+        $caExpireInterval = (new \DateInterval(sprintf('P%dY', $caExpireInYears)));
+        $caExpireInDate = (new \DateTime())->diff((new \DateTime())->add($caExpireInterval));
+
+        $this->createCa($caExpireInDate->format('%a'));
```

Comment:
Simpler alternative: `$this->createCa($caExpireInYears * 365)` (but this misses leap years)

Reply:
Good note--love this even if it's bit less simple